posts/conda-forge-maintainer/index.md

```diff
@@ -0,0 +1,28 @@
+
+### Becoming a maintainer
+
+I was working on a project and wanted to use `flask-dance`. The `flask-dance` docs were showing `v7.0.0` whereas https://anaconda.org/conda-forge/flask-dance/ showed `v2.2.0` and had not been updated in over 4 years. 
+
+I did not want to be forced to install `flask-dance` using `pip` so I went over to the conda-forge [flask-dance repository](https://github.com/conda-forge/flask-dance-feedstock) on github. I saw that there was not an active maintainer and someone tried to [become one](https://github.com/conda-forge/flask-dance-feedstock/issues/7) back in 2021 to update the repository and had received no response. 
+
+I used the [mainter docs](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user-username) to be added as a maintainer. I opened this [issue](https://github.com/conda-forge/flask-dance-feedstock/issues/10) which led to this [pull request](https://github.com/conda-forge/flask-dance-feedstock/pull/11) where I was added as a maintainer. Since it had been so long since there was an activate maintainer, I had to contact conda-forge/core and admin: ```@conda-forge/core please add me, the previous maintainer has not been active in years. @conda-forge-admin, please ping conda-forge/core```
+
+### Updating a package
+Once I got maintainer status I started the process of updating the `flask-dance-feedstock` so that I could start using it with conda. 
+
+First I opened an [issue](https://github.com/conda-forge/flask-dance-feedstock/issues/12) asking to rerender. Which let to this [pull request](https://github.com/conda-forge/flask-dance-feedstock/pull/13). I actually made a small mistake here by not bumping the build number since this was simply a rerender and NOT a new build. This is outlined [here](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-recipes) in the docs. I followed [these](https://conda-forge.org/docs/maintainer/updating_pkgs.html#archiving-feedstocks) instructions to fix my mistake with this [pull request](https://github.com/conda-forge/admin-requests/pull/817). You can now see the package marked as `broken` on [anaconda](https://anaconda.org/conda-forge/flask-dance/files). 
+
+Once the rerendering was complete I started on the process of updating the recipe for `v7.0.0`. I followed the instructions on the [maintainer docs](https://anaconda.org/conda-forge/flask-dance/files) and forked the [repository](https://github.com/rmarow/flask-dance-feedstock). This was quite simple and can be seen in [this pull request](https://github.com/conda-forge/flask-dance-feedstock/pull/15). The automated checklists make the process very easy to follow. You will see that there is also rerendering as part of this process. Once I merged the pull request I soon saw the new package for `v7.0.0` on https://anaconda.org/conda-forge/flask-dance. 
```

Comment:
I think we need a conclusion paragraph here. Perhaps something like this?

```
## Becoming a conda-forge maintainer makes releasing conda packages easy

<Here I think it would be valuable to talk about your impressions of this process. Is it hard to get started? Is it worth it? Is the documentation quality good? Is the conda-forge community supportive? Anything else?>